docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md

```diff
@@ -60,28 +60,33 @@ showAsideToc: true
 
 ## Create a new security group (optional)
 
-In order to access Yugabyte Platform from outside the AWS environment, you would need to enable access by assigning an appropriate security group to the Yugabyte Platform machine. You will at minimum need to:
+In order to access Yugabyte Platform from outside the AWS environment, you would need to enable access by assigning an appropriate security group to the server hosting Yugabyte Platform. At a minimum, you need to be able to do the following:
 
 - Access the Yugabyte Platform instance over SSH (port `tcp:22`)
 - Check, manage, and upgrade Yugabyte Platform (port `tcp:8800`)
-- View the Yugabyte Platform console (port `tcp:80`)
+- View the Yugabyte Platform UI (port `tcp:80`)
 
-If you are using your own custom VPCs (self-managed configuration), the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, 5433, and 6379. For more information on ports used by YugabyteDB, refer to [Default ports](../../../../reference/configuration/default-ports).
+If you are using your own Virtual Private Cloud (VPC) as a self-managed configuration, the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 11000, 12000, 13000, 9300, 9042, 5433, 6379. For more information on ports used by YugabyteDB, refer to [Default ports](../../../../reference/configuration/default-ports).
 
-To create a security group that enables these, go to **EC2 > Security Groups**, click **Create Security Group** and then add the following values:
+To create a security group that enables these, use the Amazon console to navigate to **EC2 > Security Groups**, click **Create Security Group**, and then add the following values:
+
+- Enter a meaningful name, such as, for example, `yugaware-sg`.
 
-- For the name, enter `yugaware-sg` (you can change the name if you want).
 - Add a description (for example, `Security group for Yugabyte Platform access`).
-- Add the appropriate IP addresses to the **Source IP ranges** field. To allow access from any machine, add `0.0.0.0/0` but note that this is not very secure.
-- Add the ports `22`, `8800`, and `80` to the **Port Range** field. The **Protocol** selected must be `TCP`. For a self-managed configuration, also add the 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, 5433, and 6379 TCP ports.
 
-You should see something like the screenshot below. Click **Create** next.
+- Add the appropriate IP addresses to the **Source IP ranges** field. To allow access from any computer, add `0.0.0.0/0` but note that this is not very secure.
```

Comment:
Don't see Source IP ranges field in screenshot - supposed to click Add Rule at some point?

Reply:
This field is called "Source". Changed.